xcodeproj/internal/opts.bzl

```diff
@@ -526,16 +572,19 @@ def _process_compiler_opts(
         *   `quotes_includes`: A `list` of quote include paths parsed.
         *   `includes`: A `list` of include paths parsed.
     """
-    conlyopts, cxxopts, search_paths = _process_copts(
+    conlyopts, cxxopts, conly_search_paths, cxx_search_paths = _process_copts(
         conlyopts = conlyopts,
         cxxopts = cxxopts,
         build_settings = build_settings,
     )
-    swiftcopts = _process_swiftcopts(
-        swiftcopts,
+    full_swiftcopts = _process_full_swiftcopts(
+        full_swiftcopts,
         package_bin_dir = package_bin_dir,
         build_settings = build_settings,
     )
+    swift_search_paths = _process_user_swiftcopts(
+        user_swiftcopts,
+    )
```

Comment:
We should probably wrap these two into a `swiftcopts, swift_search_paths = _process_swiftopts(` to mirror how `_process_copts` wraps c and c++.